### DISCUSSION TITLE
VST midi keyboard & visualizer

### DISCUSSION THREAD

**Discussion by jsalzer** [onlaj/Piano-LED-Visualizer#518] [Q&A]
I have a Hammer 88 midi keyboard that does not have any built-in sounds, it relies on VSTs like Kontakt, etc for the sounds.  This means that the VST software talks to the keyboard via midi, so within the VST software I have to choose the midi port for my Hammer 88 in order to get sound.  I built the Piano-LED-Visualizer and I want to connect it to my PC, If I were to connect the visualizer directly to the Hammer 88, I wouldn't have any sound.  Is it possible to connect the visualizer to my PC and also have my Hammer 88 talk to the VST software?  I did some reading on this and have not found a solution.  Any help is much appreciated!

**Comment by ztbishop** [comment]
There is a computer app called "midisplit" for windows and mac.  It will turn your keyboard into two virtual midi devices on your computer.  You can then assign one of them to your VST, and the other to RTPMidi for the Visualizer.
